notebooks/data-sources/gcsweb-ci/build-logs/build_log_EDA.ipynb

```diff
@@ -0,0 +1,1093 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">logs_path = "../../../../data/external/build-logs/"&nbsp; # Where to save log files
metadata_path = "../../../../data/external/"&nbsp; # Where to save metadata
</pre><p> </p><p>I think it would be better to store the sample data in <code>data/raw/gcs/build-logs/</code>  and <code>data/raw/gcs/build-metadata/</code>.  WDYT? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Makes sense. Updated.</p><p> </p>